.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: csharp
+script:

```

Comment:
Yeah, I saw that as well, but then I read:

`When the solution key is present, Travis will run NuGet package restore and build the given solution. You can also specify your own scripts, as shown in the next section.`

Which isn't required, i.e. the build process will take care of the building